Communication/Packets/Incoming/Ambassadors/SendAlertAmbassadorComposer.cs

```diff
@@ -17,6 +22,8 @@ public Task Parse(GameClient session, ClientPacket packet)
             if(habbo == null)
                 return Task.CompletedTask;
 
+            _ambassadorsManager.AddLogs(session.GetHabbo().Id, habbo.Username, "Alert");
```

Comment:
I would move the whole logic of this packet handel to the ambassadors service. Then you can create an method like `Task Warn(Habbo ambassador, Habbo target, string message)`

Reply:
This allows the method to be reused by future fomponents lile plugins.